manager/templates/default/security/login.tpl

```diff
@@ -180,7 +180,7 @@
                 <div class="c-languageselect">
                     <select name="manager_language" id="modx-login-language-select" class="c-languageselect__select" aria-label="{$_config.cultureKey}">
                         {foreach $languages as $language => $native}
-                            <option lang="{$language}" value="{$language}"{if $language == $_config.cultureKey} selected{/if}>{$native|capitalize}</option>
+                            <option lang="{$language}" data-iso-country-code="{$language}" aria-label="{$native|capitalize} selected" value="{$language}"{if $language == $_config.cultureKey} selected{/if}>{$native|capitalize}</option>
```

Comment:
Is the aria-label really necessary? And if yes, should it include the "selected" string?

Reply:
Checked in MacOS - the `lang` tag is enough for us, an additional attribute has been added `aria-label`